@ndlib/gatsby-theme-marble/src/components/Shared/ActionButtonGroup/DownloadButton/DownloadModalContent/DownloadCitation/index.js

```diff
@@ -1,11 +1,56 @@
 /** @jsx jsx */
 import PropTypes from 'prop-types'
 import { jsx } from 'theme-ui'
+import { useStaticQuery, graphql } from 'gatsby'
+import typy from 'typy'
+import sx from './sx'
 
 const DownloadCitation = ({ iiifManifest }) => {
-  return (
-    <div sx={{}}>Citation for <code>{iiifManifest.slug}</code> TBD.</div>
+  const { site } = useStaticQuery(
+    graphql`
+      query {
+        site {
+          siteMetadata {
+            siteUrl
+          }
+        }
+      }
+    `,
   )
+  let biblio = ''
+  const metadata = {}
+  if (iiifManifest.type === 'Collection') {
+    biblio = `${typy(iiifManifest, 'label.en[0]').safeString}, ${typy(iiifManifest, 'slug').safeString.replace('item/', '')}.
+      ${typy(iiifManifest, 'provider[0].homepage[0].label.en[0]').safeString}, Hesburgh Libraries, University of Notre Dame, South Bend, IN.
+      ${typy(site, 'siteMetadata.siteUrl').safeString}/${typy(iiifManifest, 'slug').safeString}`
```

Comment:
Let's throw this into a variable since we repeat it in at least two spots: 
`${typy(site, 'siteMetadata.siteUrl').safeString}/${typy(iiifManifest, 'slug').safeString}`